src/components/SearchBar.js

```diff
@@ -1,12 +1,34 @@
-import React from "react";
+import React, { Component } from "react";
+// import search from '../helpers.js';
 import "../css/SearchBar.css";
-class SearchBar extends React.Component {
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: ''
+    }
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      searchQuery: e.target.value
+    }, this.props.updateResults(this.state.searchQuery));
+  }
+
   render() {
     return (
       <section className="searchBarContainer">
         <form className="SearchBar">
-          <input type="search" placeholder="Search..." className="searchTerm" />
-          <button type="submit" className="searchButton">
+          <input 
+            onChange={this.handleChange}
+            type="search" 
+            placeholder="Search..." 
+            className="searchTerm" 
```

Comment:
Should we set the value to the `Value={this.state.searchQuery}` ??
- Just like how we saw in today's lesson..

Reply:
Great suggestion! However, I do not believe that we need the value of the search input to persist in any way since the info we need to grab will come out of the objects that the search returns.